op_crates/web/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 
 [package]
 name = "deno_web"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
@ry actually not sure if we need to release new version of `deno_web`, and... does it deploy automatically?

Reply:
It does. Let's just cut a new release even tho there are no changes. If not the crate publishing mechanism may break.

Reply:
Great 👍